puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_release.groovy

```diff
@@ -0,0 +1,74 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 4, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: "*/${packaging_branch}"]],
+                    extensions: [[$class: 'CleanCheckout']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/pulpcore-packaging']
+                    ]
+                ])
+
+            }
+        }
+        stage('Find packages') {
+            steps {
+                copyArtifacts(projectName: env.JOB_NAME, optional: true)
```

Comment:
What is getting copied in Pulpcore case?

Reply:
That's your code to copy the "commit" file from the last green job.

Reply:
Touche.